spec/views/user_index_spec.rb

```diff
@@ -3,25 +3,34 @@
 
 RSpec.feature 'UsersIndices', type: :feature do
   let(:user) do
-    User.create(
-      name: 'John Doe',
-      email: 'john.doe+2@example.com',
-      bio: "Hello, I'm John!",
-      posts_counter: 0
-    ).tap do |u|
-      u.photo.attach(io: File.open(Rails.root.join('app', 'assets', 'images', 'profile.jpg')), filename: 'profile.jpg')
-    end
+    user = User.new(name: 'John Doe', email: 'john.doe+2@example.com', bio: "Hello, I'm John!", posts_counter: 0,
+                    password: 111_111)
+    user.save
+    user
   end
 
-  def image_exists?(url)
-    response = Net::HTTP.get_response(URI.parse(url))
-    response.code == '200'
-  rescue StandardError
-    false
+  let!(:post) do
+    post = Post.create(author_id: 1, title: 'Post 6', text: 'First Time Home Buyer Tips', comments_counter: 2,
+                       likes_counter: 3, user:)
+    post.save
+    post
   end
 
-  it 'displays the user name and profile image on the index page' do
-    visit '/user'
-    expect(image_exists?(user.photo)).to be false
+  describe 'index page' do
+    it 'I can see the name of all other users.' do
+      visit '/user'
+      page.has_content?(user.name)
+    end
+
+    it 'can see the number of posts each user has written.' do
+      visit '/user'
+      page.has_content?(user.posts_counter)
+    end
+
+    it "When I click on a user, I am redirected to that user's show page." do
+      User.create(name: 'Jame', email: 'john.james+2@example.com', bio: "Hello, I'm james!", posts_counter: 3,
+                  password: 211_111)
+      visit '/user'
+    end
```

Comment:
- [ ] Let's ensure to include test for the below scenarios for _user index_ views: 
   * I can see the number of posts each user has written
   * When I click on a user, I am redirected to that user's show page